pkg/validation/microdeposits/micro_deposits.go

```diff
@@ -57,6 +59,11 @@ func createMicroDeposits(
 		micro.TransferIDs = append(micro.TransferIDs, xfer.TransferID)
 	}
 
+	// skip the debit for Savings accounts when configured to do so
+	if skipDebit(cfg.Withdraw, dest.Account) {
```

Comment:
I suggest removing `skipDebit` and being explicit:
```go
if !cfg.Withdraw.FromSavingsAccount && dest.Account.Type != customers.Savings {
 return micro, nil    
}
```

To figure out what `skipDebit` does, I had to look into its implementation.

Reply:
I'm hoping my bool logic is correct here, SKIP the debit from Savings if both match:
- Not configured to withdraw from savings, `FromSavingsAccount` is false
- account type is not `Savings`

Reply:
I would rework this part as you suggest @vxio. We already discussed it with @adamdecaf while ago as it was not clear for me as well. after our change it seems it's still unclear.

In code we should read it as this:
```
if (we should not withdraw from savings account && current account is savings)
```

what are the options to name this "we should not withdraw from savings account"?
* config.Withdraw.NotFromSavings
* config.Withdraw.SkipForSavings
* ...?

Reply:
`config.Withdraw.FromSavingsAccount` is ok since it defaults to `false` like @adamdecaf mentioned.

Something like `config.Withdraw.IsSavingsActive` could also work.

Reply:
What if we renamed `skipDebit` to `skipDebitFromSavings`?

Reply:
That could work.

Reply:
@adamdecaf maybe we can rename config option into `skipDebitFromSavings` to simplify if statement to this:
```go
if cfg.Withdraw.skipDebitFromSavings && dest.Account.Type == customers.Savings {
  ... 
}
```

Reply:
The default behavior needs to skip debits from savings accounts, so `skipDebitFromSavings bool` would have a default of `false` - that wouldn't skip debits and so I'm opposed to that naming.